### DISCUSSION TITLE
Autopilot deos not work as it should

### DISCUSSION THREAD

**Discussion by mohamad20111390** [carla-simulator/carla#6855] [Q&A]
hi everyone,

when I activate autopilot mode in simulation, cars drive off the road, especially while turning into right, or left at junctions. does anyone have faced this problem and how correct this behavior ?

thanks
mohammad

**Comment by ttgamage** [comment]
Have you tried `--sync` option? Similar issues I had with Scenario Runner were resolved when the `--sync` flag was passed to it.
